src/assets/scripts/client/ui/TutorialView.js

```diff
@@ -450,33 +438,32 @@ export default class TutorialView {
 
         this.tutorial_step({
             title: 'Bon voyage, aircraft!',
-            text: ['When the aircraft crosses the airspace boundary, it will ',
+            text: ['When the aircraft leaves your airspace , it will switch to center and',
                    'automatically remove itself from the flight strip bay on the right.',
-                   'Congratulations, you\'ve successfully taken off one aircraft.'
+                   'Congratulations, you\'ve successfully departed an aircraft.'
             ].join(' '),
             side: 'left',
             position: tutorial_position
         });
 
         this.tutorial_step({
             title: 'Arrivals',
-            text: ['Now, onto arrivals. Click on any arriving aircraft in the radar screen; after',
-                   'you\'ve selected it, use the altitude/heading/speed controls you\'ve learned in',
-                   'order to guide it to be in front of a runway. Make sure to get the aircraft down to',
-                   'around 4,000ft, and 10-15 nautical miles (2-3 range rings) away from the airport.',
-                   'While you work the airplane, read the next step.'
+            text: ['Now, onto arrivals. Click on any arriving aircraft in the radar screen; after you\'ve',
```

Comment:
We might consider moving the section on arrivals on top of departures. By the time you finish reading the section on departures you'll have aircraft all over the scope crossing the ILS and flying all over the place. Thoughts anyone? @erikquinn

Reply:
I thought so too, but I wasn't sure on how to organise it, I'll look into it now, but I'd appreciate any input!